### ISSUE TITLE
Why is there no SVC?

### ISSUE THREAD

**Issue by syedzayyan** [javascriptdata/scikit.js#258]
Hello!

Thanks for the excellent library. I noticed some SVC files but they are commented out. Is there any reason why SVC is not implemented?

Thank You!

**Comment by dcrescim** [comment]
Hey @syedzayyan! Great question. For industrial libraries like scikit-learn, they solve the SVC problem by solving the dual, and using more linear algebra tricks. These tricks aren't really supported in the current version of Tensorflow.js. Initially I attempted to solve the SVC problem by using gradient descent (without the linear algebra tricks), but ran out of time before starting a new job.

Feel free to submit a PR if this speaks to you, and I will see it merged in.
